apps/cyberstorm-storybook/LinkLibrary.tsx

```diff
@@ -14,9 +14,18 @@ const Link = (props: LinkProps): React.ReactElement => {
   const { children, queryParams, url } = props;
   const q = queryParams ? `?${queryParams}` : "";
 
-  return <a href={`${url}${q}`}>{children}</a>;
+  return (
+    <a onClick={(event) => linkOnClick(event)} href={`${url}${q}`}>
```

Comment:
`onClick={linkOnClick}` would amount to same behaviour without the wrapping function.